modules/eks/main.tf

```diff
@@ -1,8 +1,8 @@
 locals {
-  primary_role_map   = data.terraform_remote_state.primary_roles.outputs.role_name_role_arn_map
-  delegated_role_map = data.terraform_remote_state.delegated_roles.outputs.role_name_role_arn_map
-  eks_outputs        = data.terraform_remote_state.eks.outputs
-  vpc_outputs        = data.terraform_remote_state.vpc.outputs
+  primary_role_map   = module.primary_roles.outputs.role_name_role_arn_map
```

Comment:
@osterman Will have to see how this component dependency shakes out in regards to our single AWS Account tutorial idea.